xyginext/src/graphics/SpriteSheet.cpp

```diff
@@ -90,7 +91,8 @@ bool SpriteSheet::loadFromFile(const std::string& path, TextureResource& texture
             }
 
             Sprite spriteComponent;
-            spriteComponent.setTexture(*texture);
+            if (texture != NULL)
```

Comment:
nullptr please :) I personally prefer braces around the statement too,  just for consistency

Reply:
No problem, will be changed in next commit